index.html

```diff
@@ -19,7 +19,6 @@ <h3>Awesome Books</h3>
         </ul>
       </nav>
       <p id="currentDate"></p>
-
       <!-- Add a Book -->
       <section class="add-new-book">
           <h3 class="book-heading">Add a New Book</h3>
```

Comment:
- [ ] After adding books to the collection If I remove books from the collection It gets removed but when I refresh the page, I still see those books on the collection. Please make sure to fix this issue. Removing books from the collection should also update the local storage and after the page refresh the removed books should not display on the collection.

Reply:
fixed